### PR TITLE
titan-server: disable

### PR DESCRIPTION
**Pull request by chenrui333**
- [ ] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [ ] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

- https://github.com/thinkaurelius/titan/issues/1393
- https://github.com/thinkaurelius/titan/issues/1392

---

seeing 

```
  ==> Downloading http://s3.thinkaurelius.com/downloads/titan/titan-1.0.0-hadoop1.zip
  curl: (6) Could not resolve host: s3.thinkaurelius.com
  ...
```

https://github.com/Homebrew/homebrew-core/actions/runs/10837005065/job/30072042509

---

stats:

image

### CODE REVIEW

**Comment by cho-m** [comment]
In addition to deprecation, can add archive.org URLs so we can still build until we remove formula.

**Comment by chenrui333** [comment]
we can build, but I dont think artifact does not work anymore? if you see the linked issue.

**Comment by cho-m** [comment]
I wonder what is best way to apply our documentation on this:

> Deprecated formulae should continue to be maintained by the Homebrew maintainers so they still build from source and their bottles continue to work (even if unmaintained upstream). If this is not possible, they should be disabled.

Or if our deprecation policy should be updated.

**Comment by chenrui333** [comment]
I am okay to disable it directly, it also kind of applying to https://github.com/Homebrew/homebrew-core/pull/188498 as well

File: Formula/t/titan-server.rb [hunk 1]
```diff
@@ -26,6 +21,12 @@ class TitanServer < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "c6d98726b834d82fe0adb786919652c5f0e0974ff3cb03969b2c69042cd4998c"
   end
 
+  # upstream is not responsive on the issues and no commits since 2015 dec
+  # community has forked the project to janusgraph
+  # https://github.com/thinkaurelius/titan/issues/1393
+  # https://github.com/thinkaurelius/titan/issues/1392
+  disable! date: "2024-09-12", because: :unmaintained
```
Comment (carlocab) [inline Formula/t/titan-server.rb:28]:
```suggestion
  disable! date: "2025-01-01", because: :unmaintained
```
Let's have a short deprecation period here.

**Review by carlocab — APPROVED** [review]

**Pull request merged**